shard.yml

```diff
@@ -1,5 +1,5 @@
 name: action-controller
-version: 3.5.1
+version: 4.0.0
```

Comment:
Should this just be a minor bump as it’s a big fix?

Reply:
it technically changes behaviour